### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by atxbyea**
Include instructions for migrating from venv to docker

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ All images for the `octoprint/octoprint` image are multi-arch images, and we pub
       - [Container Environment based configs](#container-environment-based-configs)
       - [Restarting OctoPrint](#restarting-octoprint)
       - [Editing Config files manually](#editing-config-files-manually)
+      - [Migration From Venv](#migration-from-venv)
```
Comment (LongLiveCHIEF) [inline README.md:38]: let's put this in the extended docs instead of in the README.

File: README.md [hunk 2]
```diff
@@ -124,6 +125,26 @@ docker-compose stop config-editor && docker-compose rm config-editor
...
+#### Migration from venv
+
+If you are currently running in a venv it is easy to migrate to a docker container by copying the existing venv data to a docker mapped persistent volume.
+
+e.g
+
+```
+cp -r /home/octoprint/.octoprint/. /docker/octoprint
+```
+
```
Comment (LongLiveCHIEF) [inline README.md:137]: the current documentation all encourages the use of docker volumes instead of host mounted volumes.  Your example is the first place in the docs where a host-mounted volume is used, so it makes sense that your example code should include a command to make that directory.

Technically, there's no reason to even do a copy if you're using host-mounted volumes, since there's no reason to move the files.

Either way, it should probably include a bit of an example for a `docker-compose.yml` file.

File: README.md [hunk 3]
```diff
@@ -124,6 +125,26 @@ docker-compose stop config-editor && docker-compose rm config-editor
...
+Remember to stop and disable the service for your venv
+
+```
+systemctl stop octoprint.service
+systemctl disable octoprint.service
+```
+
+Then map your copied folder to the right location inside the container (by default this is /octoprint/octoprint/) and start the container.
```
Comment (LongLiveCHIEF) [inline README.md:145]: the venv will have plugins, which exist in the container at /octoprint/plugins.

File: README.md [hunk 4]
```diff
@@ -124,6 +125,26 @@ docker-compose stop config-editor && docker-compose rm config-editor
...
+Remember to also map the serial \ usb device as a persistent name, not dynamic names
```
Comment (LongLiveCHIEF) [inline README.md:146]: I have an article coming up about cgroups, udev rules, and more.  For now, I recommend striking this line and we can add something back to it like remember to configure your video devices (see more here link)

**Review by LongLiveCHIEF — CHANGES_REQUESTED** [review]
I love the fact that you're willing to contribute, and love the fact that you're willing to contribute to documentation _even more_.

There are a few things I think this needs before it can be pulled, and I left comments in the specific places where they come up.

Overall though, my present line of thinking is that the best way to migrate from native install to docker install would be:

1. to use the native "create backup", 
2. shut down native service, 
3. spin up docker and docker volume, and 
4. import backup

Since it's docker in addition to octoprint, there's a dozen different ways to do this, but what we need to watch out for is that there are several different paths where python libs get installed to in the container (another enhancement incoming to detail that), and a straight copy might not work for everyone.

That being said, a straight copy would have the same footpriint as the v2 octo container filesytem, so perhaps enabling `AUTOMIGRATE=true` would be an essential part of doing your copy method proposed.

TL;DR:

I think this is an excellent and essential document. I want to be able to run my test rig through your instructions before I merge it, but right now it breaks in a few places when I follow your steps.

**Comment by atxbyea** [comment]
Sorry @LongLiveCHIEF I have been incredibly busy with work, I will get back to you hopefully this week 👍
